pkg/apis/kops/cluster.go

```diff
@@ -199,6 +199,9 @@ type ClusterSpec struct {
 
 	// ClusterAutoscaler defines the cluster autoscaler configuration.
 	ClusterAutoscaler *ClusterAutoscalerConfig `json:"clusterAutoscaler,omitempty"`
+
+	// Azure specifies the configuration specific to Azure.
+	Azure *AzureSpec `json:"azure,omitempty"`
```

Comment:
This one would go into `CloudConfiguration`, I think. Together with the `OpenstackConfiguration` struct.

Reply:
I see. Moved!